Lib/uuid.py

```diff
@@ -572,22 +572,24 @@ def _netstat_getnode():
 try:
     import _uuid
     _generate_time_safe = getattr(_uuid, "generate_time_safe", None)
+    _has_stable_extractable_node = _uuid.has_stable_extractable_node
```

Comment:
```suggestion
    _has_stable_extractable_node = getattr(_uuid, "has_stable_extractable_node", False)
```

This failed downstream, I think you need to use `getattr` as above.

Reply:
(though maybe something else weird is going on)

Reply:
Hum, that's weird. The _uuid module should unconditionally have the attribute.

Reply:
That's what I thought from reading the patch too. I didn't dig deeper though, since this worked around the failing test.